### ISSUE TITLE
Not clear what NonRestrictive means

### ISSUE THREAD

**Issue by lvanvugt** [MicrosoftDocs/dynamics365smb-devitpro-pb#933]
Not clear what NonRestrictive means. What makes it different from Disabled?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5e9a88b1-f7e9-e4cb-6b06-4d9e11ef4e35
* Version Independent ID: c9b7c384-2643-75e6-c9cd-308d3b77042f
* Content: [TestPermissions Property - Business Central](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/properties/devenv-testpermissions-property)
* Content Source: [dev-itpro/developer/properties/devenv-testpermissions-property.md](https://github.com/MicrosoftDocs/dynamics365smb-devitpro-pb/blob/live/dev-itpro/developer/properties/devenv-testpermissions-property.md)
* Service: **dynamics365-business-central**
* GitHub Login: @blrobl
* Microsoft Alias: **SusanneWindfeldPedersen**

**Comment by lvanvugt** [comment]
This issue is hanging now for almost 5 month. When is this going to be picked up?

**Comment by lvanvugt** [comment]
@SusanneWindfeldPedersen could you please give an update?

**Comment by lvanvugt** [comment]
While working on the [2nd edition of my test automation book](https://www.packtpub.com/product/automated-testing-in-microsoft-dynamics-365-business-central/9781789804935) I cleared it for myself, but I still think this topic could be improved on this.
This is how I have described the 3 main values of the TestPermissions property:

- **Disabled**: Based on this value the test runner should not implement any permission restrictions.
- **NonRestrictive**: Based on this value the test runner could implement a broad set of user permissions.
- **Restrictive**: Based on this value the test runner could implement a restricted set of user permissions. Note that this is the default value.
